data/vijaysheru.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Vijay Sheru",
+  "type": "personal",
+  "bio": "Mechanical Enginner turned Software Developer-Arts,Design,Artificial Intelligence,Code,Cryto,Cloud"
+  "avatar": "https://github.com/vijaysheru",
```

Comment:
We no longer need the avatar key. So please remove it from your `json`. 

```suggestion
```

Reply:
Changes are done. Thanks